docs/hopsworksai/aws/eks_ecr_integration.md

```diff
@@ -274,8 +258,9 @@ Click *Save rules* to save the updated rules to the security group.
 
 ## Step 5: Allow Hopsworks.ai to delete ECR repositories on your behalf
 
-You need to add another inline policy to your role or user connected to Hopsworks.ai, see [../getting_started.md].
-First, navigate to [AWS management console](https://console.aws.amazon.com/iam/home#), then click on *Roles* or *Users* depending on which connection method you have used in Hopsworks.ai, and then search for your role or user name and click on it.  Go to the *Permissions* tab, click on *Add inline policy*, and then go to the *JSON* tab. Paste the following snippet, click on *Review policy*, name it, and click *Create policy*.
+For hopsworks.ai to be able to clean up the ECR repo when terminating your hopsworks cluster, you need to add new inline policy to the [Cross-Account role](getting_started.md#option-1-using-aws-cross-account-roles) or [user connected to Hopsworks.ai](getting_started.md#option-2-using-aws-access-keys), that you set up when [connecting your AWS account to hopsworks.ai](getting_started.md#step-1-connecting-your-aws-account).
+
+Navigate to [AWS management console](https://console.aws.amazon.com/iam/home#), then click on *Roles* or *Users* depending on which connection method you have used in Hopsworks.ai, and then search for your role or user name and click on it.  Go to the *Permissions* tab, click on *Add inline policy*, and then go to the *JSON* tab. Paste the following snippet, click on *Review policy*, name it, and click *Create policy*.
```

Comment:
Copy the following snippet, click on *Review policy*, name it, and click *Create policy*, then paste the snippet in the policy field.

Reply:
What you wrote does not really match what needs to be done. I replaced the "Past the following snippet" with the same `Replace the existing JSON permissions with the JSON permissions below` as in your comment above on a similar situation.